### PR TITLE
Added Superblocks Lab IDE to the generic doc section

### PR DESCRIPTION
**Pull request by javier-tarazaga**
### Description

Simply added Superblocks Lab to the generic IDE section documentation for reference inside the resources.rst

### CODE REVIEW

File: docs/resources.rst [hunk 1]
```diff
@@ -21,6 +21,8 @@ Solidity Integrations
 
 * Generic:
 
+    * `Superblocks Lab <https://lab.superblocks.com/>`_
+        Browser-based IDE. Built in browser blochain VM, Metamask integration (one click deployments to Testnet/Mainnet), transaction logger and WebApp live coding using with Web3.
```
Comment (chriseth) [inline docs/resources.rst]: The syntax is a little off towards the end. Also note that this list is sorted alphabetically - could you please move it to the end?

**Review by chriseth — CHANGES_REQUESTED** [review]
Thanks for adding!

**Comment by erak** [comment]
@javier-tarazaga Are you still working on this? Otherwise we could take this over :)

File: docs/resources.rst (hunk 1 above)
Comment (javier-tarazaga) [reply docs/resources.rst]: Hi @chriseth yes of course! Let me do the changes :)

Comment (axic) [reply docs/resources.rst]: "blochain" seems to be a typo

**Review by axic — APPROVED** [review]

**Comment by Sacricarl** [comment]
Javier just had his first newborn baby and is taking time off, I'm sure he is fine with you handling this :)

**Comment by erak** [comment]
@Sacricarl Done :)

**Pull request merged**